services/web/client/source/class/qxapp/io/rest/ResourceFactory.js

```diff
@@ -87,35 +87,35 @@ qx.Class.define("qxapp.io.rest.ResourceFactory", {
       const basePath = qxapp.io.rest.ResourceFactory.API;
 
       // Singular resource
-      var project = new qxapp.io.rest.Resource({
-        // Retrieve project
+      const study = new qxapp.io.rest.Resource({
+        // Retrieve study
         get: {
           method: "GET",
           url: basePath+"/projects/{project_id}"
         },
 
-        // Update project
+        // Update study
         put: {
           method: "PUT",
           url: basePath+"/projects/{project_id}"
         },
 
-        // Delete project
+        // Delete study
         del: {
           method: "DELETE",
           url: basePath+"/projects/{project_id}"
         }
       });
 
       // Plural resource
-      var projects = new qxapp.io.rest.Resource({
-        // Retrieve list of projects
+      const studies = new qxapp.io.rest.Resource({
+        // Retrieve list of studies
         get: {
           method: "GET",
           url: basePath+"/projects?type=user"
```

Comment:
ah this starts to be very confusing... study or project or studies or projects?... ;)